functions.php

```diff
@@ -24,6 +24,10 @@ function _s_setup() {
 		 */
 		load_theme_textdomain( '_s', get_template_directory() . '/languages' );
 
+		add_theme_support( 'amp', array(
+			'comments_live_list' => true,
+		) );
```

Comment:
Naturally this shouldn't be merged as-is into `master`. It would need to be opt-in by the theme developer.